### PR TITLE
new yaml format

### PR DESCRIPTION
**Pull request by LawlAoux**

### CODE REVIEW

File: config_to_bot/resolvers.py [hunk 1]
```diff
@@ -342,17 +288,74 @@ async def remote_function(params: Dict[str, Any]):
...
+            ),
+        )
+
+    return listen_to_amount_of
+
+
+def _listen_to_intent(intent: Tuple[str, ...]):
     return gamla.pipe(
-        listen,
+        agenda.consumes_external_event(_listen_to_type("intent")(intent)),
```
Comment (uriva) [inline config_to_bot/resolvers.py:318]: move `intent` to be the first line of the `pipe`

Comment (LawlAoux) [reply config_to_bot/resolvers.py:318]: intent is inside consume external so I can't do that...

File: config_to_bot/examples/pizza/remoteFunctions.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 const express = require("express");
 const app = express();
+const renderToppings = (toppings) => {
+  if (toppings != `none`) {
```
Comment (uriva) [inline config_to_bot/examples/pizza/remoteFunctions.js:4]: won't it be better to send this as an array? (where none is `[]`)

**Review by uriva — APPROVED** [review]

File: config_to_bot/resolvers.py (hunk 1 above)
Comment (uriva) [reply config_to_bot/resolvers.py:318]: `pipe(intent, _listen_to_type("intent"), agenda.consumes_external_event, ...`

**Review by uriva — APPROVED** [review]

**Pull request merged**